Week-11/redux-assignments/src/App.test.js

```diff
@@ -1,8 +1,52 @@
-import { render, screen } from '@testing-library/react';
+import { render, screen ,fireEvent} from '@testing-library/react';
 import App from './App';
 
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
-});
+test('Main switching between assignments button should be rendered',()=>{
+    render(<App/>)
+    const buttonEle = screen.getByRole('button',{name:/switch to/i})
+    expect(buttonEle).toBeInTheDocument()
+})
+
+test('Room component heading should be rendered',()=>{
+    render(<App/>)
```

Comment:
instead of rendering each time we can make use of hooks beforeeach and move this code there..